src/Purger/ORMPurger.php

```diff
@@ -248,7 +246,7 @@ private function getTableName(ClassMetadata $class, AbstractPlatform $platform):
 
     /** @param ManyToManyOwningSideMapping|mixed[] $assoc */
     private function getJoinTableName(
-        $assoc,
+        ManyToManyOwningSideMapping|array $assoc,
```

Comment:
I'll rather revert this change, since it happens in a private method, changing it later will not be a breaking change.